pkg/conversion/deep_copy_test.go

```diff
@@ -106,3 +107,34 @@ func TestDeepCopyPointerSeparate(t *testing.T) {
 		t.Errorf("deep copy wasn't deep: %#q %#q", x, y)
 	}
 }
+
+var result interface{}
+
+func BenchmarkDeepCopy(t *testing.B) {
+	table := []interface{}{
+		map[string]string{},
+		int(5),
+		"hello world",
+		struct {
+			A, B, C struct {
+				D map[string]int
+			}
+			X []int
+			Y []byte
+		}{},
+	}
+	f := fuzz.New().RandSource(rand.NewSource(1)).NilChance(.5).NumElements(0, 100)
+	for i := range table {
+		out := table[i]
+		obj := reflect.New(reflect.TypeOf(out)).Interface()
+		f.Fuzz(obj)
+		table[i] = obj
+	}
```

Comment:
minor nit-- fuzzing isn't cheap, you'll probably get slightly more accurate times if you reset the timer here.